deploy/helm-chart/templates/deployment.yaml

```diff
@@ -101,6 +101,9 @@ spec:
           mountPath: /etc/nginx/secrets
         - name: nginx-run
           mountPath: /var/run/nginx
+        {{- with .Values.nginxGateway.extraVolumeMounts -}}
```

Comment:
@kate-osborn Did you see the same issue here as with the other PR where `with` needed to be changed to `if`?

Reply:
No, `with` works fine in this case since there are existing volumes and volume mounts.